lib/backend/database/helper.dart

```diff
@@ -6,7 +6,7 @@ import 'package:files/backend/providers.dart';
 class EntityStatCacheHelper {
   Future<EntityStat> get(String path) async {
     final EntityStat? stat =
-        isar.entityStats.where().pathEqualTo(path).findFirstSync();
+        isar.entityStats.where().pathNotEqualTo(path).findFirstSync();
```

Comment:
why this logic change? i checked the generated file and the pathEqualTo function exists, did i get this wrong originally?

Reply:
I apologize, somehow i made this a breaking change. correcting this.
The reason might be I am not aware of `isar` package fully.